integrations/scylladb/prometheus_metadata.yaml

```diff
@@ -37,6 +37,10 @@ platforms:
         prometheus_name: scylla_storage_proxy_coordinator_read_timeouts
         kind: CUMULATIVE
         value_type: DOUBLE
+      - name: prometheus.googleapis.com/scylla_storage_proxy_coordinator_writes_coordinator_outside_replica_set/counter
```

Comment:
Can you:
- add a `version: '1'` for this platform?
- update the `characteristic_metric` to be a metric that is *not* collected by the 5.2 version of the exporter? This will allow us to differentiate between the two versions of this integration.